src/views/certifications/components/signup-form-area/components/newsletter-signup-form/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+import MarketoForm from '@hashicorp/react-marketo-form'
+import type { MarketoForm as MarketoFormProps } from '@hashicorp/react-marketo-form/types'
+import { safeAnalyticsTrack } from 'lib/analytics'
+import SubmitMessage from './submit-message'
+import s from './marketo-form-data.module.css'
+
+export interface NewsletterSignupFormProps {
+	marketoFormData: MarketoFormData
+	buttonText?: string
+	// Note: we've hacked around styles, so only 'dark' is supported
+	appearance?: 'dark'
+}
+
+export interface MarketoFormData {
+	id: number
+	form: MarketoFormProps
+}
+
+function NewsletterSignupForm({
+	buttonText = 'Submit',
+	appearance = 'dark',
+	marketoFormData,
+}: NewsletterSignupFormProps) {
+	const [submissionStatus, setSubmissionStatus] = useState(null)
+
+	const onSubmitSuccess = () => {
+		setSubmissionStatus('success')
+		safeAnalyticsTrack('Newsletter Signup', {
+			placement: 'Dev dot certifications',
+		})
+	}
+
+	const onSubmitError = () => {
+		setSubmissionStatus('error')
```

Comment:
Maybe we can add some logging here to let us know if it's broken?

Reply:
We [fire off a Zapier to Slack webhook](https://github.com/hashicorp/react-components/blob/main/packages/marketo-form/server/api-routes.ts#L123) in the same code path that will call `onSubmitError` if that happens to be sufficient for y'all!

Reply:
Ah yeah, great call - I think I can drop in a DataDog call here or something 🤔

Reply:
Oh yes, that gets the job done! Thanks for that context @dstaley !

Reply:
oops, jynx! that's good to know. would that identify Dev Dot specifically at all? Or to put it another way, if the form started failing on Dev Dot, but not Dot Com, would we be able to tell those apart?

Reply:
yeah, we get the current page URL in the response, so we can distinguish between them!